src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -76,6 +81,10 @@ public Loan(JsonObject representation, Item item, User user, User proxy,
       representation.put("proxyUserId", proxy.getId());
     }
 
+    if (loanPolicy != null) {
```

Comment:
If the loan keeps a reference to a loan policy object, why do we need to update the state separately?

Reply:
For consistency reasons: when a loan is created, it will have the loanpolciy name and id in its representation.

Reply:
What aspect are you trying to be consistent with? The updating of the `userId` or `proxyId`, for example?

Reply:
Agree, that it can be safely removed

Reply:
I removed setting the loan policy name